packages/next-css/css-loader-config.js

```diff
@@ -78,13 +84,14 @@ module.exports = (
   }
 
   const cssLoader = {
-    loader: isServer ? 'css-loader/locals' : 'css-loader',
+    loader: 'css-loader',
```

Comment:
should keep `loader: isServer ? 'css-loader/locals' : 'css-loader',`

Reply:
Reason? css-loader 2.0 has `exportOnlyLocals`

Reply:
@radeno I didn't have time to dig into it but after upgrade I found the CSS modules stylesheet disappeared on the server rendering page. 
Then I revert this line and everything works again, meaning the CSS modules are back.

Reply:
@chuyik then check what is changing. This module just wraps `css-loader`
https://github.com/webpack-contrib/css-loader/releases

and you can see just first sence: `css modules are disabled by default` so you need to follow next-css readme https://github.com/zeit/next-plugins/tree/master/packages/next-css#with-css-modules

If something doesnt work then open an issue.